src/utils.jl

```diff
@@ -1,5 +1,130 @@
 using StackOverflow
 
+"""
```

Comment:
@Akshat-mehrotra I would move this to answer.jl since it's doesn't serve much value outside of that file.